.github/workflows/ci.yml

```diff
@@ -20,37 +20,36 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: ${{ matrix.node-version }}
+      - run: cd javascript
       - run: npm install
       - run: npm run build
       - run: npm run lint
 
-
   build-rust:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v3.1.0
-    - name: Install Rust Stable toolchain
-      uses: actions-rs/toolchain@v1
-      with:
-        profile: minimal
-        toolchain: stable
-        override: true
-
-    - name: Run cargo fmt
-      run: cargo fmt --all -- --check
+      - uses: actions/checkout@v3.1.0
+      - name: Install Rust Stable toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          profile: minimal
+          toolchain: stable
+          override: true
 
-    - uses: actions-rs/clippy-check@v1
-      with:
-        token: ${{ secrets.GITHUB_TOKEN }}
-        args: --all-features
+      - name: Run cargo fmt
+        run: cargo fmt --all -- --check
 
-    - name: Build
-      run: cargo build --verbose
+      - uses: actions-rs/clippy-check@v1
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          args: --all-features
 
-    - name: Run tests
-      run: cargo test --verbose --all-targets --all-features
+      - name: Build
+        run: cargo build --verbose
```

Comment:
```suggestion
        run: cargo build 
```

Reply:
Thank you. CI will need many changes (e.g. separate CI of JS with cargo and also add directories